adk/manifest.py

```diff
@@ -0,0 +1,83 @@
+import os
+import json
+import hashlib
+
+
+class ManifestData(object):
+    def __init__(self, client, model_manifest_path):
+        self.manifest_lock_path = model_manifest_path
+        self.manifest_data = get_manifest(self.manifest_lock_path)
+        self.client = client
```

Comment:
This is something I'm curious as to your thoughts about @kennydaniel; as this is able to automatically download models for users, we need a client object - and since the ADK is not actually dependent on the Algorithmia Client - we need an external implementation passed into the constructor. I think this form is reasonable, but a little complex for new users.. This client object is only required if the user is using a model manifest.